internal/controllers/s3bucket/suite_test.go

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Why not remove this file if it's not used?

Reply:
Sure. I'll remove it.

Reply:
Done.